examples/tp/hello_tp.py

```diff
@@ -6,15 +6,15 @@
 # following terms and conditions apply:
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero Public License version 3 as
```

Comment:
correct me if I'm wrong, but I think we've switched to AGPL from GPL, so the license lines should stay.